pages/Configuring/Binds.md

```diff
@@ -113,60 +128,78 @@ e -> repeat, will repeat when held.
 m -> mouse, see below
 ```
 
-## Mouse Binds
+Example Usage:
+
+```ini
+# Example volume button that allows press and hold
+binde=, XF86AudioRaiseVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+
+
+# Example volume button that will activate even while an input inhibitor is active
+bindl=, XF86AudioLowerVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-
+
+# Start wofi on hold and kill it on release
+bindr=SUPER, SUPER_L, exec, pkill wofi || wofi
+
+# See Mouse Binds section for bindm usage
+```
+
+# Mouse Binds
+
 Mouse binds are binds that heavily rely on a mouse, usually its movement.
 They will have one less arg, and look for example like this:
 
-```
+```ini
 bindm=ALT,mouse:272,movewindow
 ```
 
 this will create a bind with <key>ALT</key> + <key>LMB</key> to move the window
 with your mouse.
 
-*Available mouse binds*:
+_Available mouse binds_:
 
 | Name | Description |
 | -----|------------ |
-| movewindow | moves the active window |
+| movewindow   | moves the active window   |
 | resizewindow | resizes the active window |
 
-*Common mouse buttons' codes:*
-```
+_Common mouse buttons' codes:_
+
+```txt
 LMB -> 272
 RMB -> 273
 ```
 
-*for more, you can of course use `wev` to check.*
+_for more, you can of course use `wev` to check._
 
 {{< hint type=tip >}}
 Mouse binds, despite their name, behave like normal binds. You are free to use
 whatever keys / mods you please. When held, the mouse function will be activated.
 {{< /hint >}}
 
-## Binding mods
+# Binding mods
 
 You can bind a mod alone like this:
 
 ```ini
 bindr=ALT,Alt_L,exec,amongus
 ```
 
-## Global Keybinds
+# Global Keybinds
+
 Yes, you heard this right, Hyprland does support global keybinds for ALL apps,
 including OBS, Discord, Firefox, etc.
 
-See the `pass` dispatcher for keybinds.
+See the [`pass` dispatcher](./Dispatchers/#dispatchers) for keybinds.
 
-e.g.:
+Let's take OBS as example: the "Start/Stop Recording" keybind is set to 
+<key>SUPER</key> + <key>F10</key>, and you want to make it work globally.
 
-I've set the "Start/Stop Recording" keybind in OBS to <key>SUPER</key> +
-<key>F10</key>, and I want it to be global.
+Simply, just add
```

Comment:
either simply or just.

Reply:
This was fixed by me, you're not the only one who can speak english.
Blame @0x4nt for fucking up.